clusterloader2/testing/load/config.yaml

```diff
@@ -90,7 +97,7 @@
 
 name: load
 namespace:
-  number: {{AddInt $namespaces $schedulerThroughputNamespaces}}
+  number: {{MinInt .Nodes (AddInt $namespaces $schedulerThroughputNamespaces)}}
```

Comment:
Not sure If I understand this line. Wouldn't be better to make sure the `$namespaces` and `schedulerThroughputNamespaces` are just set properly in all the cases?

Reply:
I added some calculations below so it's easier to make sense of the change.

Without MinInt:
```
# 100 nodes
$MIN_PODS_IN_SMALL_CLUSTERS := 500
$totalSchedulerThroughputPods := MaxInt (MultiplyInt 2 $MIN_PODS_IN_SMALL_CLUSTERS) (MultiplyInt 2 .Nodes) # 1000
$schedulerThroughputPodsPerDeployment := .Nodes # 100
$schedulerThroughputNamespaces := DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment # 10
number: AddInt $namespaces $schedulerThroughputNamespaces # 11
```

```
# 1 node
$MIN_PODS_IN_SMALL_CLUSTERS := 500
$totalSchedulerThroughputPods := MaxInt (MultiplyInt 2 $MIN_PODS_IN_SMALL_CLUSTERS) (MultiplyInt 2 .Nodes) # 1000
$schedulerThroughputPodsPerDeployment := .Nodes # 1
$schedulerThroughputNamespaces := DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment # 1000
number: AddInt $namespaces $schedulerThroughputNamespaces # 1001 <- !!!
```

With MinInt:
```
# 100 nodes
same as above
number: MinInt .Nodes (AddInt $namespaces $schedulerThroughputNamespaces) # 11
```

```
# 1 node
same as above
number: MinInt .Nodes (AddInt $namespaces $schedulerThroughputNamespaces) # 1
```